src/functions/sendCandidateResults/framework/adapter/config/config-adapter.ts

```diff
@@ -42,6 +43,26 @@ export class ConfigAdapter implements IConfigAdapter {
     this.welshLetterFailTemplateId = this.getFromEnvThrowIfNotPresent('NOTIFY_POST_WELSH_FAIL_TEMPLATE_ID');
   }
 
+  private setApiKey(): void {
+    if (this.isLocal) {
+      this.apiKey = this.getFromEnvThrowIfNotPresent('NOTIFY_API_KEY');
+      return;
+    }
+
+    const secretsmanager = new awsSdk.SecretsManager();
+    const params: awsSdk.SecretsManager.GetSecretValueRequest = {
+      SecretId: 'mes-nonprod-api-documents-service', // TODO: Confirm that this is really the name of the secret
+    };
+    secretsmanager.getSecretValue(params, (err, data) => {
```

Comment:
no `.promise()` available on `.getSecretValue()`?

Reply:
yes, there is

Reply:
be good to use `await` then to stay consistent

Reply:
changed it